src/main/java/org/opentripplanner/transit/model/network/TripPatternBuilder.java

```diff
@@ -0,0 +1,105 @@
+package org.opentripplanner.transit.model.network;
+
+import java.util.BitSet;
+import java.util.Map;
+import org.opentripplanner.model.Timetable;
+import org.opentripplanner.transit.model.framework.AbstractEntityBuilder;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+@SuppressWarnings("UnusedReturnValue")
+public final class TripPatternBuilder
+  extends AbstractEntityBuilder<TripPattern, TripPatternBuilder> {
+
+  private BitSet services;
+  private Route route;
+  private StopPattern stopPattern;
+  private Timetable scheduledTimetable;
+  private String name;
+  private Map<FeedScopedId, Integer> serviceCodes;
+
+  private boolean createdByRealtimeUpdate;
+
+  private TripPattern originalTripPattern;
+
+  public TripPatternBuilder(FeedScopedId id) {
```

Comment:
```suggestion
  TripPatternBuilder(FeedScopedId id) {
```

Reply:
I fixed this for all Entities where it was public